src/components/Controls.vue

```diff
@@ -54,8 +54,9 @@
 			</div>
 			<div class="board-action-buttons">
 				<Popover>
-					<Actions slot="trigger" :style="filterOpacity" :title="t('deck', 'Apply filter')">
-						<ActionButton icon="icon-filter" />
+					<Actions slot="trigger" :title="t('deck', 'Apply filter')">
+						<ActionButton v-if="isFilterActive" icon="icon-filter_set" />
```

Comment:
I don't know `vue` that good, but won't this call and execute the method on each single change detection?

Wouldn't it be smarter in this case to use a boolean property to check against and set actively change that boolean property when a user adds tags, users or due to the list of filters (or, analog, removes them)?

Reply:
isFilterActive is a computed property so the result is cached. [This line](https://github.com/nextcloud/deck/pull/1887/files#diff-3c5f6c58f8c054bd6d60375762579633L222) is basically evaulated every time the filter is updated but the v-if condition is only triggered if the actual result of `isFilterActive` is changing.

Reply:
Well then: forget it and thanks for the explanation :)